src/04_printing.py

```diff
@@ -12,6 +12,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print('x is %s, y is %s, z is "%s"' % (x, y, z))
```

Comment:
you can do this to convert a string to a float and limit it to two decimal places 

```print("x is {}, y is {:.2f}, z is \"{}\"".format(x, y, z))```